doc/image_processing/basics.rst

```diff
@@ -0,0 +1,54 @@
+Basics
+------
```

Comment:
Any reason you don't follow GIL's markup for headings?

GIL (should) follows markup for sections recommended by [Python guide for documenting](https://devguide.python.org/documenting/#sections) and equivalent [Sphinx reStructuredText Primer](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections), that is:

```
=, for sections
-, for subsections
^, for subsubsections
", for paragraphs
```

However, in future, this may and we may decide to distinguish chapters and parts as well.is intentional.